src/msg/async/AsyncConnection.cc

```diff
@@ -233,7 +233,6 @@ ssize_t AsyncConnection::_try_send(bool more)
 
 // Because this func will be called multi times to populate
 // the needed buffer, so the passed in bufferptr must be the same.
-// Normally, only "read_message" will pass existing bufferptr in
```

Comment:
i think @yuyuyu101 is referencing the behavior in `Pipe::read_message()` in simple msgr. and he wanted to emphasis the difference between how async msgr and simple msgr read data from the wire. the former do readahead, and is stateful, the latter simply set the message with a newly (or "existing") allocated bufferptr. the latter is stateless. in this sense, this comment still applies. but it could be confusing.